package.json

```diff
@@ -15,11 +15,11 @@
   "dependencies": {
     "@google-cloud/storage": "^1.7.0",
     "foam2": "git://github.com/foam-framework/foam2.git#mdn-confluence",
-    "mdn-browser-compat-data": "git://github.com/mdn/browser-compat-data.git",
-    "web-api-confluence-dashboard": "git://github.com:mdittmer/confluence.git#mdn",
+    "mdn-browser-compat-data": "0.0.38",
+    "web-api-confluence-dashboard": "git://github.com/mdittmer/confluence.git#mdn",
```

Comment:
Why `.com/` here and `.com:`below (in the `peerDependencies`)?

Reply:
Because I submitted the PR prematurely.